internal/pkg_web/golden_test/BUILD.bazel

```diff
@@ -0,0 +1,20 @@
+load("@build_bazel_rules_nodejs//:index.bzl", "pkg_web")
+load("//internal/golden_file_test:golden_file_test.bzl", "golden_file_test")
+
+package(default_visibility = ["//visibility:public"])
+
+pkg_web(
+    name = "pkg",
+    srcs = [
+        "index.js",
+    ],
+    substitutions = {
+        "THIS_SHOULD_BE_REPLACED": "THIS_SHOULD_BE_REPLACED_WITH_THIS",
```

Comment:
nit: I know this is a substitutions, but at first I didn't see the different between the strings after the build as only the suffix changed 😄 
Perhaps provide a real world example of a replacement here, as this is mostly centered around version replacement.

Reply:
I agree, a bit confusing. we can add an example that replaces with an arbitrary key from the volatile file therefore we would have tests and examples for both use cases.